requirements.txt

```diff
@@ -1,3 +1,6 @@
 git+https://github.com/italia/docs-italia-theme.git
 git+https://github.com/pdavide/sphinxcontrib-discourse
+git+https://github.com/pdavide/sphinxcontrib-versiontag
 doc8
+sphinx
+recommonmark
```

Comment:
Se non fail build di file markdown, ma solo rst, `recommonmark` non è necessario.

Reply:
Ok.

Reply:
Il conf.py contiene comandi in `recommonmark`. Lasciarlo non nuoce, giusto?